react/react.d.ts

```diff
@@ -2117,6 +2117,12 @@ declare namespace __React {
         item(index: number): Touch;
         identifiedTouch(identifier: number): Touch;
     }
+
+    //
+    // React internal functions called in JSX generation.
+    //
+
+    function __spread( target: any, ...sources: any[] ): any;
```

Comment:
Please use consistent style:
no space after `(`
no space before `)`

I also think `{}` or `Object` would be preferable to `any`, but I concede that I don't really know how this functionality works.